wwwroot/.placeholder

```diff
@@ -1 +0,0 @@
-��
```

Comment:
Why did you delete this file?

Reply:
As long as we push the contents of the _wwwroot_ directory a placeholder is unnecessary. Also the directory was recreated by Gulp even if you deleted it, so it was double pointless.